examples/SampleApp/.node-version

```diff
@@ -0,0 +1 @@
+18
```

Comment:
why 18? 

docusaurus will break I think

Reply:
but no problem in CI though.. locally will be a problem

Reply:
Ideally we update the docusaurus repo to support 18

Reply:
> Ideally we update the docusaurus repo to support 18

That's right, in the meanwhile, we should all use nvm to manage those versions.